### PR TITLE
Ticket 06 layout event list

### PR DESCRIPTION
**Pull request by sarayfg**
HI @ifuenteromero !
In this branch I fix the card list layout, specially in desktop.
It's done with grid.
As the img is background, I create two div, one for the image and the other one for the info, both of them have a 50% height.

Let me know if you have any doubt.

### CODE REVIEW

**Comment by ifuenteromero** [comment]
Ok, I think it's everything ok up to 1400px. 

- Check color font in title-event is #1e0a3c darkBlue

- and in hover is #3659e3 $blueKing

**Comment by sarayfg** [comment]
Hi @ifuenteromero I fixed the font color and add the different color when hover.

I add the tablet version too.
I erased the mixin that includes flex in the body.

Let me know any doubt or problem

File: src/components/EventCard/index.js [hunk 1]
```diff
@@ -29,11 +29,13 @@ const EventCard = props => {
...
-      style={{ backgroundImage: `url(${eventImage})` }}
     >
-      <p className="card__event-date">{formatedDate}</p>
-      <h2 className="card__event-name">{eventName}</h2>
-      <p className="card__event-place">{`${eventPlace}, Madrid`}</p>
+      <div className="img__container" style={{ backgroundImage: `url(${eventImage})` }}></div>
+      <div className="info__container">
+        <p className="card__event-date">{formatedDate}</p>
+        <h2 className="card__event-name">{eventName}</h2>
+        <p className="card__event-place">Madrid</p>
```
Comment (irenegf83) [inline src/components/EventCard/index.js]: `<p className="card__event-place">{`${eventPlace}, Madrid`}</p>`

Comment (sarayfg) [reply src/components/EventCard/index.js]: Thanks!! solved!

**Pull request merged**